### ISSUE TITLE
best_checkpoint_metric is accuracy even for finetuning?

### ISSUE THREAD

**Issue by PussyCat0700** [facebookresearch/av_hubert#107]
Hi,

I found that best_checkpoint_metric is accuracy even in finetuning config file, which goes:

>https://github.com/facebookresearch/av_hubert/blob/258fb50e155134eec2c4b49c2ae8de267075fd18/avhubert/conf/finetune/large_lrs3_433h.yaml#L15

Due to highly encapsulated design of fairseq, I am having a hard time finding out how this metric is working in finetuning.  I can understand that accuracy refers to avhubert label classification in pretraining (though I don't know where it is computed in code implementation), but I don't understand what accuracy stands for in context of finetuning. And probably, WER seems to be a better best_checkpoint_metric in finetuning?

**Comment by chevalierNoir** [comment]
The accuracy is the proportion of correctly predicted next tokens given all previous ground-truth tokens on the decoder side. We haven't tried WER but it should be better than accuracy in choosing the checkpoint.

**Comment by PussyCat0700** [comment]
> The accuracy is the proportion of correctly predicted next tokens given all previous ground-truth tokens on the decoder side. We haven't tried WER but it should be better than accuracy in choosing the checkpoint.

Many thanks!
